std/random.d

```diff
@@ -1542,8 +1559,16 @@ if ((isIntegral!(CommonType!(T1, T2)) || isSomeChar!(CommonType!(T1, T2))) &&
 
 /**
 Generates a uniformly-distributed number in the range $(D [T.min,
-T.max]) for any integral type $(D T). If no random number generator is
-passed, uses the default $(D rndGen).
+T.max]) for any integral or character type $(D T). If no random
+number generator is passed, uses the default $(D rndGen).
+
+Params:
+    urng = (optional) random number generator to use;
+           if not specified, defaults to $(D rndGen)
+
+Returns:
+    Random variate uniformly distributed across all the different
```

Comment:
I would really rather not -- my preference is very much for the `std.random` documentation to use precise statistical terminology as much as possible.  Do you think it's likely to be that confusing for users?

Tho' I suppose for precision's sake I should probably write, `Random variate drawn from the uniform distribution` ... :-)